boms/cloud-oss-bom/RELEASING.md

```diff
@@ -89,24 +101,29 @@ new release is available on Maven Central.
       (no PR required)
     * https://github.com/googleapis/google-http-java-client/blob/master/docs/setup.md
     * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
+* In google3 run:
+    * `$ scripts/update_docs.sh <old version> <new version>`
+      * For example, `$ scripts/update_docs.sh 16.3.0 16.4.0`
+      * When asked whether to add changes to the first CL, answer "y".
+    * Sanity check the CL and send it for review. Use "Suggest reviewers" to pick a reviewer.
```

Comment:
"Suggest reviewers" was failing for me. Maybe say here who should be contacted?

Reply:
"Suggest reviewers" not working is strange. But I'll add the account names after moving to an internal site.